.devcontainer/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:22.04
+
+# Install python
+RUN apt-get update
+RUN apt-get install -y \
+    python3 \
+    python3-venv \
+    python3-pip
+
+# Dev tools require an additional update
+RUN apt-get update
+RUN apt-get install -y \
+    git \
+    gdb \
+    curl \
+    zip \
+    unzip \
+    tar \
+    autoconf \
+    libtool \
+    libtool-bin \
+    pkg-config
+
+WORKDIR /tools
+
+# Create new venv at container root and install libs
+RUN python3 -m venv /tools/venv
+ENV PATH /tools/venv/bin:$PATH
+RUN pip install --upgrade pip setuptools wheel
+RUN pip install \
+    cmake \
+    ninja \
+    pybind11 \
+    numpy \
+    pytest \
+    pytest-cov
+
+# FIXME make conditional on arm, s390x, ppc64le and riscv
+ENV VCPKG_FORCE_SYSTEM_BINARIES=1
```

Comment:
Upon checking the documentation, this just forces VCPKG not to install things like CMake and Ninja. This probably doesn't hurt anything.

Reply:
Ah, sorry, I saw your last reply after I'd made a change! I've made it conditional so only sets this when `uname -m` is aarch64. I'll leave it in though, as I would prefer not to set build flags unnecessarily; now if one is working on a Linux container they can be sure no extra flags are being set.